.travis.yml

```diff
@@ -28,8 +28,10 @@ env:
   - DB=postgres
 
 install:
-  - sh -e ci/install.sh
+  - sh -e ci/apache/install.sh
```

Comment:
It's actually still using basic auth in the apache tests. Doing the full matrix would be ideal, but I think I'd like to have that separate discussion about CI first; as you've pointed out, running times are getting bad, and I'm wanting a strategy for keeping that under control.